ydb/core/viewer/json_handlers_viewer.cpp

```diff
@@ -204,7 +204,7 @@ void InitViewerStorageUsageJsonHandler(TJsonHandlers &handlers) {
 }
 
 void InitViewerClusterJsonHandler(TJsonHandlers& handlers) {
-    handlers.AddHandler("/viewer/cluster", new TJsonHandler<TJsonCluster>(TJsonCluster::GetSwagger()), 4);
+    handlers.AddHandler("/viewer/cluster", new TJsonHandler<TJsonCluster>(TJsonCluster::GetSwagger()), 5);
```

Comment:
magic numbers :)

Reply:
> magic numbers :)

versions :) for UI to know what to expect from the current backend